providers/openfeature-provider-flagd/pyproject.toml

```diff
@@ -37,6 +37,9 @@ dependencies = [
   "coverage[toml]>=6.5",
   "pytest",
   "pytest-bdd",
+  "testcontainers",
+  "asserts",
+  "grpcio-health-checking==1.60.0",
```

Comment:
needed to pin this to some version, as there was an ` Detected incompatible Protobuf Gencode/Runtime versions when loading grpc_health/v1/health.proto: gencode 5.27.2 runtime 5.27.1. Runtime version cannot be older than the linked gencode version.` error